plugins/vbe/README.md

```diff
@@ -0,0 +1,326 @@
+# Verified Boot for Embedded (VBE)
+
+## Introduction
+
+This plugin is used for boards which use the VBE system. This allows the
+platform to be updated from user space.
+
+## Firmware Format
+
+Firmware updates are held within a CAB file, a Windows format which allows
+files to be collected and compressed, similar to a tar file. The CAB file can be
+created using the `gcab` tool, as shown in the VBE `example` directory.
+
+Inside the CAB file is the firmware itself, in Flat Image Tree (FIT) format.
+This is typically called `firmware.fit` and can be creatd by the `mkimage` tool,
+or using device tree tools such as `dtc` and `fdtput`.
+
+The FIT supports multiple configuration, each of which is intended to update a
+particular board type. This allows the same firmware update to be used for a
+number of related boards, including sharing certain images, at the expense of
+increasing the update size.
+
+A typical FIT file looks like this:
+
+```fdt
+/ {
+    timestamp = <0x62a74c6c>;
+    description = "Firmware image with one or more FDT blobs";
+    creator = "U-Boot mkimage 2021.01+dfsg-3ubuntu0~20.04.4";
+    #address-cells = <0x00000001>;
+    images {
+        firmware-1 {
```

Comment:
This looks dangerously close to JSON, no?

Reply:
Few quotes and support for binary data and fast parsing:

https://en.wikipedia.org/wiki/Devicetree#Device_Tree_formats